src/Promotions/InPluginUpsells/resources/js/donation-options.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ *
+ * @unreleased
+ *
+ */
+declare global {
+    interface Window {
+        GiveLegacyFormEditor: GiveLegacyFormEditorData;
+    }
+}
+
+export interface GiveLegacyFormEditorData {
+    apiRoot: string;
+    apiNonce: string;
+}
+
+/**
+ *
+ * @unreleased
+ *
+ */
+const recurringProductRecommendation = document.querySelector('.givewp-donation-options');
+const dismissAction = document.querySelector('.givewp-donation-options_close');
+
+function updateContainerPlacement() {
+    const preceedingContent = document.querySelector('._give_custom_amount_text_field');
+    preceedingContent.insertAdjacentElement('afterend', recurringProductRecommendation);
+}
+
+updateContainerPlacement();
```

Comment:
Resolved: https://github.com/impress-org/givewp/pull/6802/commits/77d08c765255f69dab60c11c2b151651a8f76fab

Updated to validate elements prior to handling to avoid console errors https://github.com/impress-org/givewp/pull/6802/commits/e9ecc0e6475f93a3e12c8cdf2e1433840ca0f17f